main.go

```diff
@@ -32,13 +62,33 @@ func main() {
 
 	log.Infof("\nRunning go test...")
 
+	packageCodeCoveragePth, err := packageCodeCoveragePath()
+	if err != nil {
+		failf(err.Error())
+	}
+
+	codeCoveragePth, err := codeCoveragePath()
+	if err != nil {
+		failf(err.Error())
```

Comment:
+1

Reply:
i designed the method to report a proper error message. no need for wrapping the error